ui/apps/platform/src/Containers/Policies/Modal/ImportPolicyJSONModalError.tsx

```diff
@@ -88,6 +92,7 @@ function ImportPolicyJSONError({
                             <DuplicatePolicyForm
                                 updateResolution={updateResolution}
                                 showKeepBothPolicies={showKeepBothPolicies}
+                                allowOverwriteOption={allowOverwriteOption}
```

Comment:
_technically_ this has nothing to do with the multi import but we should've had this check regardless so i think it's fine to leave it